### PR TITLE
add missed err check in lib/client/client.go when try to read stdout

### PR DESCRIPTION
**Pull request by miolini**
add missed err check in lib/client/client.go when try to read stdout

### CODE REVIEW

**Comment by kontsevoy** [comment]
miolini: wouldn't this lead to permanent `io.EOF` errors?

**Comment by miolini** [comment]
Yes, sure.

File: lib/client/client.go [hunk 1]
```diff
@@ -93,17 +93,23 @@ func (proxy *ProxyClient) GetSites() ([]services.Site, error) {
...
 	}()
 
 	if err := proxySession.RequestSubsystem("proxysites"); err != nil {
 		return nil, trace.Wrap(err)
 	}
 	select {
-	case <-done:
+	case doneErr, ok := <-done:
+		if ok && doneErr != nil {
+			return nil, trace.Errorf("read err: %s", doneErr)
```
Comment (klizhentas) [inline lib/client/client.go]: this hides the original error. We have `trace.Wrap(err)` for this

Comment (miolini) [inline lib/client/client.go]: How I could specify a error context with trace.Wrap?

Comment (kontsevoy) [inline lib/client/client.go]: It infers it automatically from its call stack. Just call `trace.Wrap(err)` - we always wrap errors. And it's ok to double or triple-wrap (it's smart enough to not do that)

Comment (miolini) [inline lib/client/client.go]: Got it. Thank you.

**Comment by miolini** [comment]
@kontsevoy Please, take a look.

**Comment by kontsevoy** [comment]
@miolini same as above: these aren't actionable errors and the extra code doesn't add much. I agree with you on one thing: there should be a clean way to detect lost connections and reconnects.

**Pull request closed without merging**